pkg/systemd/host.js

```diff
@@ -108,16 +110,23 @@ function ServerTime() {
      * behavior and formatting of a Date() object in the absence of
      * IntlDateFormat and  friends.
      */
-    Object.defineProperty(self, 'now', {
+    Object.defineProperty(self, 'utc_fake_now', {
```

Comment:
It's explained in the multi-line comment above that line ... you have no idea how hard doing server time in an application like Cockpit is. It's the sort of time I wish I had never had to spend time on ... but covering all the cases here took days of figuring out ... while reviewing and merging the "Set Server Time" code. This is a place where the integration tests were *so* hard ... yet *so* revealing.